build-test.xml

```diff
@@ -4983,6 +4983,30 @@ tell application "Safari" to quit
 		</sequential>
 	</macrodef>
 
+	<macrodef name="prepare-test-environment-variables">
+		<attribute name="test.environment.variables" />
```

Comment:
@Kvale1733 you can just move the logic into the parent macro. There doesn't seem to a be a point to nest it into a macro here. Its probably more messy to create an extra target than to have a wrapper here